### ISSUE TITLE
Not working from subview

### ISSUE THREAD

**Issue by jeanbaptistebeau** [BeeMeeMan/FullScreenCover#2]
It doesn't have one of the most useful features of `.fullScreenCover` in my opinion, which is to be able to call it from a subview that is not full screen. I've been trying to replicate that without success for the past few months :(

**Comment by BeeMeeMan** [comment]
Yes it can't do that, but in most cases it's not a problem, here's an example of how I use it in such cases on projects. I will think about how to write the second part where it will be possible to call it from any place.

TabView(selection: $selectedTab) {
            ForEach(TabBarItem.allCases, id:\.rawValue) { tabItem in
                NavigationView {
                        switch selectedTab {
                        case .home:
                            HomeView()
                        case .edit:
                            EditView()
                                .environmentObject(filterManager)
                        case .gallery: GalleryView()
                    }
                }
                .environmentObject(coordinator)
                .navigationViewStyle(.stack)
            }
        }
        .easyFullScreenCover(isPresented: $coordinator.showFullScreen) {
            Group {
                switch coordinator.fullScreenState {
                case .fullImage:
                    FullScreenImageView()
                case .filterEdit:
                    FilterView()
                }
            }
            .environmentObject(coordinator)
            .environmentObject(galleryManager)
        }
